airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -29,7 +35,10 @@ type FullTableProps = CreditConsumptionByConnector & {
   destinationIcon?: string;
 };
 
-const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ creditConsumption }) => {
+export const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ creditConsumption }) => {
+  const isBillingInsightsEnabled = useExperiment("billingPage.billingInsights", false);
+  const { workspaceId } = useCurrentWorkspace();
+
   const query = useQuery<{ sortBy?: string; order?: SortOrderEnum }>();
```

Comment:
I know not related to the changes in this PR, but maybe we could type the expected query params for this route and let them live in `BillingPage.tsx` (as e.g. `BillingPageQueryParams`). That way if we add/remove more query params in this route, we can re-use the same type? Not at all blocking, just occurred to me while reading.

Reply:
Yep!  Though, I am not 100% sure we'll continue to use the query params when we switch over to the new return type from the API. 

I'm going to have to noodle on that.